Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py

```diff
@@ -1521,6 +1522,717 @@ def url_analyses_feed_from_date_output(response, start_time):
+        )
+        markdown = f"{markdown}\n {threats_table}"
+
+    third_party = response_json.get("rl").get("third_party_reputations")
+    if third_party:
+        third_party_statistics = third_party.get("statistics")
+
+        markdown = f"""{markdown}\n ### Third party statistics\n **CLEAN**: {third_party_statistics.get("clean")}
+        **MALICIOUS**: {third_party_statistics.get("malicious")}
+        **UNDETECTED**: {third_party_statistics.get("undetected")}
+        **TOTAL**: {third_party_statistics.get("total")}
+        """
+
+        sources_table = tableToMarkdown(
+            name="Third party sources",
+            t=third_party.get("sources"),
+        )
+        markdown = f"{markdown}\n {sources_table}"
+
+    files_statistics = response_json.get("rl").get("downloaded_files_statistics")
+    markdown = f"""{markdown}\n ### Downloaded files statistics\n **KNOWN**: {files_statistics.get("known")}
+    **MALICIOUS**: {files_statistics.get("malicious")}
+    **SUSPICIOUS**: {files_statistics.get("suspicious")}
+    **UNKNOWN**: {files_statistics.get("unknown")}
+    **TOTAL**: {files_statistics.get("total")}
+    """
+
+    dbot_score = Common.DBotScore(
+        indicator=domain,
+        indicator_type=DBotScoreType.DOMAIN,
+        integration_name="ReversingLabs TitaniumCloud v2",
+        score=0,
+        reliability=RELIABILITY
+    )
+
+    indicator = Common.Domain(
+        domain=domain,
+        dbot_score=dbot_score
+    )
+
+    results = CommandResults(
+        outputs_prefix="ReversingLabs",
+        outputs={"domain_report": response_json},
+        readable_output=markdown,
+        indicator=indicator
+    )
+
+    return results
+
+
+def domain_downloaded_files_command():
+    domain_ti = create_domain_ti_object()
+
+    domain = demisto.getArg("domain")
+    classification = demisto.getArg("classification")
+    limit = int(demisto.getArg("result_limit"))
+    per_page = int(demisto.getArg("results_per_page"))
+
+    try:
+        response = domain_ti.get_downloaded_files_aggregated(
+            domain=domain,
+            classification=classification,
+            results_per_page=per_page,
+            max_results=limit
+        )
+    except Exception as e:
+        return_error(str(e))
+
+    results = domain_downloaded_files_output(response=response, domain=domain)
+    return_results(results)
+
+
+def domain_downloaded_files_output(response, domain):
+    files_table = tableToMarkdown(
+        name="Downloaded files",
+        t=response
+    )
+
+    markdown = f"## ReversingLabs Files downloaded from domain {domain}\n {files_table}"
+
+    dbot_score = Common.DBotScore(
+        indicator=domain,
+        indicator_type=DBotScoreType.DOMAIN,
+        integration_name="ReversingLabs TitaniumCloud v2",
+        score=0,
+        reliability=RELIABILITY
+    )
+
+    indicator = Common.Domain(
+        domain=domain,
+        dbot_score=dbot_score
+    )
+
+    results = CommandResults(
```

Comment:
the same as https://github.com/demisto/content/pull/30507#discussion_r1376014643

Reply:
Same answer.